services/client-api/index.js

```diff
@@ -1,23 +1,24 @@
 const express = require('express');
-const responseBuilder = require('./responseBuilder');
+const { folder, board, group, task, teammember } = require('./models');
```

Comment:
Please note that all of the different handlers you defined on the client-api (such as folder, board, group, task, teammember) are not models but rather handlers. Models are the database units defined in the infrastructure/db folder. Perhaps you should change it to a more descriptive name? such as handlers?